alacritty_terminal/src/term/mod.rs

```diff
@@ -2819,8 +2823,11 @@ mod tests {
 
         term.goto(Line(1), Column(1));
 
-        // NOTE We want to test what damage information is being produced by each function
-        // individually and the entire damage that we could get from `[Term::damage]`.
+        // NOTE While we can use `[Term::damage]` to access terminal damage information, in the
+        // following tests we will be accessing `term.damage.lines` directly to avoid adding extra
+        // damage information(like cursor and Vi cursor) which we're not testing in the following
+        // cases.
```

Comment:
```suggestion
        // NOTE While we can use `[Term::damage]` to access terminal damage information, in the
        // following tests we will be accessing `term.damage.lines` directly to avoid adding extra
        // damage information (like cursor and Vi cursor), which we're not testing.
```

Already has a `in the following tests` in the beginning, best not to repeat ourselves. Also tiny parenthesis mistake.